pkg/networkmanager/ip-settings.jsx

```diff
@@ -133,8 +133,8 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
         });
 
         // Prevent dialog from closing because of <form> onsubmit event
-        if (event)
-            event.preventDefault();
```

Comment:
Wow, how did that ever compile? I don't see any `event` identifier anywhere.

Reply:
It seems the browser and eslint treat `event` as a super global, it's really meh

Reply:
Ah, so like `window`? It's `undefined` in my browser console, but it could still mean that eslint etc. treat it as a global. Thanks!